tools/docker/tests/test_flows.sh

```diff
@@ -35,25 +42,86 @@ report() {
 
 start_gw_repeater() {
     dbg "delete running containers before starting test"
-    docker rm -f $(docker ps -a -q) >/dev/null 2>&1
+    docker rm -f gateway
+    docker rm -f repeater
     dbg "start gw and repeater"
-    ${scriptdir}/test_gw_repeater.sh >/dev/null 2>&1
+    ${scriptdir}/test_gw_repeater.sh -d 5 #>/dev/null 2>&1
```

Comment:
Please don't leave commented-out code in the commit - or when you do, make sure that you document in the commit message why you have commented-out code.